.github/policies/issue.remove-needs-author-feedback.yml

```diff
@@ -14,8 +14,11 @@ configuration:
       - hasLabel:
           label: 'Needs: Author Feedback'
       then:
-      - addLabel:
-          label: 'Needs: Attention :wave:'
+      - if:
+        - isOpen
+        then:
+        - addLabel:
+            label: 'Needs: Attention :wave:'
```

Comment:
If an author closed an issue with a comment, we were still flagging it as needs attention. This should prevent that.